### PR TITLE
Chart: make `--max-requests-inflight` and `--max-mutating-requests-inflight` configurable

### PR DESCRIPTION
**Pull request by pigletfly**
Signed-off-by: pigletfly <wangbing.adam@gmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: charts/templates/karmada_apiserver.yaml [hunk 1]
```diff
@@ -70,6 +70,8 @@ spec:
             - --requestheader-username-headers=X-Remote-User
             - --tls-cert-file=/etc/kubernetes/pki/karmada.crt
             - --tls-private-key-file=/etc/kubernetes/pki/karmada.key
+            - --max-requests-inflight=1500
+            - --max-mutating-requests-inflight=500
```
Comment (RainbowMango) [inline charts/templates/karmada_apiserver.yaml:74]: Just a question, can users configure this parameter?

leave to owners @jrkeen @algebra2k

Comment (pigletfly) [reply charts/templates/karmada_apiserver.yaml:74]: ok,we can have a default value,then make them configurable

Comment (jrkeen) [reply charts/templates/karmada_apiserver.yaml:74]: > ok,we can have a default value,then make them configurable

I agree with you

**Review by RainbowMango — COMMENTED** [review]
/retitle Chart: make `--max-requests-inflight` and `--max-mutating-requests-inflight` configurable

**Comment by pigletfly** [comment]
@jrkeen PTAL

**Comment by jrkeen** [comment]
> @jrkeen PTAL

image

@pigletfly Do you think the `--enable-priority-and-fairness` configuration needs to be configured?

Configuration comment:
image

**Comment by pigletfly** [comment]
I don't think so.Because it's not enabled by default,and if it's enabled, the `--max-requests-inflight` and `--max-mutating-requests-inflight` flags are still useful,ref :https://kubernetes.io/docs/concepts/cluster-administration/flow-control/#priority-levels

**Pull request merged**